services/efs-guardian/src/simcore_service_efs_guardian/_meta.py

```diff
@@ -0,0 +1,65 @@
+""" Application's metadata
+
+"""
+
+from importlib.metadata import distribution, version
+from importlib.resources import files
+from pathlib import Path
+from typing import Final
+
+from models_library.basic_types import VersionTag
+from packaging.version import Version
+from pydantic import parse_obj_as
+
+_current_distribution = distribution("simcore-service-efs-guardian")
+__version__: str = version("simcore-service-efs-guardian")
+
+
+APP_NAME: Final[str] = _current_distribution.metadata["Name"]
+API_VERSION: Final[str] = __version__
+VERSION: Final[Version] = Version(__version__)
+API_VTAG: Final[VersionTag] = parse_obj_as(VersionTag, f"v{VERSION.major}")
+RPC_VTAG: Final[VersionTag] = parse_obj_as(VersionTag, f"v{VERSION.major}")
+
+
+def get_summary() -> str:
+    return _current_distribution.metadata.get_all("Summary", [""])[-1]
+
+
+SUMMARY: Final[str] = get_summary()
+PACKAGE_DATA_FOLDER: Final[Path] = Path(f'{files(APP_NAME.replace("-", "_")) / "data"}')
+
+# https://patorjk.com/software/taag/#p=display&f=ANSI%20Shadow&t=Elastic%20file%0Asystem%20guardian
+APP_STARTED_BANNER_MSG = r"""
```

Comment:
OMG I thought that @sanderegg 's banners where the most crazy ones but I see he has competition! :-D

Reply:
😱 <- my actual face when seeing this

Reply:
😍

Reply:
pretty much looking forward! I'll keep it as a surprise